Drop CAN from nRF9280

gmarull opened the pull request:
CAN is not ready yet for this SoC, so remove it for now.

Fixes #85065

gmarull committed:
boards: nrf9280pdk: remove can enablement

CAN is not supported yet, so remove it from the board for now.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>

gmarull committed:
dts: nordic: nrf9280: drop can120/121 nodes

CAN is not ready yet for this platform (clock control drivers). So
remove the nodes for now.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>

kartben reviewed (changes requested):
drop `can` from `supported` in `boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280_cpuapp.yaml` please

fabiobaltieri commented:
`/__w/zephyr/zephyr/tests/kernel/gen_isr_table/src/main.c:35:2: error: #error "Target not supported"`

does this needs an exclude in `tests/kernel/gen_isr_table/testcase.yaml`? looks like targets need explicit support

fabiobaltieri commented:
Hey, looking at this again, I think it may indeed also need an exclude on that test in `tests/kernel/gen_isr_table/testcase.yaml`, maybe just

```diff
--- a/tests/kernel/gen_isr_table/testcase.yaml
+++ b/tests/kernel/gen_isr_table/testcase.yaml
@@ -72,6 +72,7 @@ tests:
     platform_exclude:
       - m2gl025_miv
       - adp_xc7k/ae350
+      - nrf9280pdk/nrf9280/cpuppr
     filter: CONFIG_RISCV_PRIVILEGED
     extra_configs:
       - CONFIG_GEN_IRQ_VECTOR_TABLE=y
```

@gmarull can you look into it?

gmarull committed:
tests: kernel: gen_isr_table: exclude nrf9280pdk/nrf9280/cpuppr

First, SoC is not supported, but when I tried to add it, build results
in RAM overflow of ~10K. Excluding the platform for now, so CI is happy.

Signed-off-by: Gerard Marull-Paretas <gerard.marull@nordicsemi.no>

fabiobaltieri commented on tests/kernel/gen_isr_table/testcase.yaml:
```diff
@@ -72,6 +72,7 @@ tests:
     platform_exclude:
       - m2gl025_miv
       - adp_xc7k/ae350
+      - nrf9280pdk/nrf9280/cpuppr
```
ugh, it's still running it, does this need to be `nrf9280pdk/nrf9280/cpuppr/xip`?
